ports/stm32/boards/stm32h7a3.ld

```diff
@@ -0,0 +1,42 @@
+/*
+    GNU linker script for STM32H7A3
+*/
+
+/* Specify the memory areas */
+MEMORY
+{
+    FLASH (rx)      : ORIGIN = 0x08000000, LENGTH = 2048K   /* 8K sector size */
+    FLASH_ISR (rx)  : ORIGIN = 0x08000000, LENGTH = 32K     /* bootloader */
+    FLASH_FS (r)    : ORIGIN = 0x08008000, LENGTH = 96K     /* minimal file system */
```

Comment:
The flash page erase size is 128k, so the bootloader must take that much space, and the minimum filesystem must also be 128k.

Reply:
This is probably related to #11138 for the adapted flash block size on the H7A and H7B devices